resources/v2/02_cabs.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE cabs (
+    id INTEGER NOT NULL PRIMARY KEY,
+    cab_type ENUM('Sedan', 'Hatchback', 'Auto', 'Bike'),
+    cab_number integer NOT NULL UNIQUE,
+    cab_model TEXT NOT NULL,
+    driver_id INTEGER NOT NULL,
+    created_at timestamp WITH time zone DEFAULT NOW(),
+    updated_at timestamp WITH time zone,
+    deleted_at timestamp WITH time zone
+);
```

Comment:
This query does not work.   
Create the enum type first then assign it to the table  
https://www.postgresql.org/docs/current/datatype-enum.html

Reply:
changed